skstore/ts/src/prv/skstore_impl.ts

```diff
@@ -84,7 +86,7 @@ class EHandleImpl<K extends TJSON, V extends TJSON> implements EHandle<K, V> {
     const mapperObj = new mapper(...params);
     Object.freeze(mapperObj);
     if (!mapperObj.constructor.name) {
-      throw new Error("The class must have a name.");
+      throw new Error("Mapper classes must be named.");
```

Comment:
Is what we really want `Mapper classes must be defined at top-level` and being named is just a way to approximately check that? I wonder if a user seeing this error for an attempt to use a nested class would react by trying to add a name property to their class.